201-servicebus-private-endpoint/README.md

```diff
@@ -0,0 +1,96 @@
+
+This sample demonstrates how to create a Linux Virtual Machine in a virtual network that privately accesses a Service Bus namespace using an [Azure Private Endpoint](https://docs.microsoft.com/en-us/azure/private-link/private-endpoint-overview). Azure Private Endpoint is a network interface that connects you privately and securely to a service powered by Azure Private Link. Private Endpoint uses a private IP address from your virtual network, effectively bringing the service into your virtual network. The service could be an Azure service such as Azure Storage, Azure Cosmos DB, SQL, etc. or your own Private Link Service. For more information, see [What is Azure Private Link?](https://docs.microsoft.com/en-us/azure/private-link/private-link-overview). For more information on the DNS configuration of a private endpoint, see [Azure Private Endpoint DNS configuration](https://docs.microsoft.com/en-us/azure/private-link/private-endpoint-dns).
+
+## Architecture ##
+
+The following picture shows the architecture and network topology of the sample.
+
+![Architecture](images/architecture.png)
+
+The ARM template deploys the following resources:
+
+- Virtual Network: this virtual network has a single subnet that hosts an Linux (Ubuntu) virtual machine
+- Network Security Group: this resource contains an inbound rule to allow the access to the virtual machine on port 22 (SSH)
+- The virtual machine is created with a managed identity which is assigned the contributor role at the resource group scope level
+- A Public IP for the Linux virtual machine
+- The NIC used by the Linux virtual machine that makes use of the Public IP
+- A Linux virtual machine used for testing the connectivity to the storage account via a private endpoint
+- A Log Analytics workspace used to monitor the health status of the Linux virtual machine
+- A Blob Storage Account used to store data on the Azure Data Lake File System
+- A Service Bus namespace
+- A Private DNS Zone for Blob private endpoints
+- A Private DNS Zone for Service Bus private endpoints
+- A Private Endpoint to let the virtual machine access the Blob Storage Account via a private address
+- A Private Endpoint to let the virtual machine access the messaging entities in the Service Bus namespace via a private address
+
+The ARM template uses the [Azure Custom Script Extension](https://docs.microsoft.com/en-us/azure/virtual-machines/extensions/custom-script-linux) to download and run the following Bash script on the virtual machine. The script performs the following steps:
+
+- Validates the parameters received by the Custom Script extension
+- Updates the system and upgrade packages
+- Runs the nslookup command against the public URL of the Storage Account to verify that this gets resolved to a private address
+- Runs the nslookup command against the public URL of the Service Bus namespace to verify that this gets resolved to a private address
+
+**servicebus_nslookup.sh**
+
+```bash

```

Comment:
can we just link to the script instead of duplicating inline?  Will create a maint issue otherwise

Reply:
Sounds good. Fixed.